src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -3723,6 +3767,21 @@ .maxstack  2
         verifier2.VerifyIL("Program.Test", testIL);
         verifier2.VerifyIL("Extensions.M2", m2IL);
 
+        var remove = """
+    .custom instance void [mscorlib]System.Runtime.CompilerServices.ExtensionAttribute::.ctor() = (
+        01 00 00 00
+    )
+""";
+        comp2 = CreateCompilationWithIL(src2, expectedTypeIL.Remove(expectedTypeIL.IndexOf(remove), remove.Length));
```

Comment:
Consider also testing that `CreateCompilationWithIL(src2, expectedTypeIL)` works (to make sure that there is not some other problem with the IL compilation and we are really testing what we intend to).